env_files/back-safe.env

```diff
@@ -18,6 +18,7 @@ SMTP_ADDRESS=mailcatcher
 SMTP_PORT=1025
 AWS_EMBEDDINGS_REGION=eu-central-1
 AWS_TOXICITY_DETECTION_REGION=eu-central-1
+HEATMAP_CALCULATION_START_HOUR_UTC=21
```

Comment:
Starting from 9 pm and going for 10 hours should always mean this happens outside working hours.